src/framework/DOCRunnable.ts

```diff
@@ -173,6 +173,7 @@ export class DOCRunnable extends AbstractRunnable {
   protected _getDebugParamsInner(childrenToRun: readonly Readonly<DOCTest>[], breakOnFailure: boolean): string[] {
     const execParams: string[] = this._getRunParamsCommon(childrenToRun);
     execParams.push('--reporters=console');
+    if (!breakOnFailure) execParams.push('--no-breaks');
```

Comment:
How about 
```
execParams.push('--no-breaks=' + breakOnFailure ? 'true' : 'false');
```
?

Reply:
I was just preservative. What if the user has a version that doesn't support that feature and it causes an error? He could then at least work around it but no setting breakOnFailure to false. Anyways, I just tested doctest 2.0.0 and even that version has the feature, so should be no problem do make the change that you are suggesting.

Reply:
Good thinking